packages/framework/src/shared/models.ts

```diff
@@ -70,6 +70,7 @@ export interface EnsembleAPIModel {
   url?: string;
   method: "GET" | "POST" | "PUT" | "PATCH";
   headers?: { [key: string]: string | number | boolean };
+  cacheExpiry?: number;
```

Comment:
Let's rename this to `cacheExpirySeconds` to make it clear what the unit of measurement is.